BREAKING.md

```diff
@@ -36,6 +37,9 @@ Additionally, `codeDetails` which was already deprecated before is now marked as
 ### `readOnly` and `readOnlyPermissions` removed from `Container`
 The `readOnly` and `readOnlyPermissions` properties from `Container` in `container-loader` was deprecated in 0.35, and has now been removed. To replace its functionality, use `readOnlyInfo` by accessing `readOnlyInfo.readonly` and `readOnlyInfo.permissions` respectively.
 
+### `maxMessageSize` removed from `container-definitions` and `driver-definitions`
+The `maxMessageSize` property from `container-definitions` and `driver-definitions` was deprecated in 0.51, and has now been removed. To replace its functionality, use `serviceConfiguration.maxMessageSize`.
```

Comment:
please clarify it's the `maxMessageSize` property on the `IConnectionDetails` interface in the packages

Reply:
(please also keep the containing packages)